File: src/nx-workspace/apps/bookstore/src/app/authors/author.ts
import { Book } from 'libs/books-ui/src/lib/books/book';

//import { Book } from '@nx-playground/books-ui/Book';

//import { Book } from '@nx-playground/books-ui';

export interface Author {
  id: string;
  name: string;
  books: Book[];
}

File: src/api/Playground.Data/Author.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace Playground.Data
{
    public class Author
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public IEnumerable<Book> Books { get; set; }
    }
}
